test/suites/migration.sh

```diff
@@ -21,6 +21,15 @@ test_migration() {
   lxc_remote move l1:nonlive l2:
   lxc_remote config show l2:nonlive/snap0 | grep user.tester | grep foo
 
+  # test `lxd import`
+  if [ "${LXD_BACKEND}" = "zfs" ]; then
+    lxc_remote init testimage backup
+    lxc_remote snapshot backup
+    sqlite3 "${LXD_DIR}/lxd.db" "DELETE FROM containers WHERE name='backup'"
```

Comment:
Oh and you're also not removing the snapshot from the database, so the test above should pass even if the container wasn't re-imported. You'll want to delete "backup/snap0" from all the DB tables too.